_ml-commons-plugin/ml-dashboard.md

```diff
@@ -0,0 +1,67 @@
+
+As of OpenSearch 2.6, you can only upload models using the API. For more information about how to upload a model to your cluster, see [Upload model to OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework#upload-model-to-opensearch)
+
+## Enabling Machine Learning in Dashboards
+
+In OpenSearch 2.6, Machine Learning is disabled by default. To enable it, you need to edit the configuration in `opensearch_dashboards.yml` and then restart your cluster.
+
+To enable the feature:
+
+1. In your OpenSearch cluster, navigate to your Dashboards home directory; for example, in Docker, `/usr/share/opensearch-dashboards`.
+2. Open your local copy of the Dashboards configuration file, `opensearch_dashboards.yml`. If you don't have a copy, get one from GitHub: [`opensearch_dashboards.yml`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml).
+3. Add the setting `ml_commons_dashboards.enabled:`  to `opensearch_dashboards.yml`. Then. set it to  `ml_commons_dashboards.enabled: true` and save the configuration file.
+4. Restart the Dashboards container.
+5. Verify that the feature configuration settings were created and configured properly by launching OpenSearch Dashboards. The Machine Learning section should appear under **OpenSearch plugins**.
+
+## Accessing Machine Learning in Dashboards
+
+To access Machine Learning in OpenSearch Dashboards,select **OpenSearch plugins** > **Machine Learning**. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/ml/ml-dashboard/ml-dashboard.png" alt="Machine Learning section in OpenSearch dashboards">
+
+Inside the Machine Learning section, you now have access to the **Deployed models** dashboard.
+
+## Deployed models dashboard
+
+The model health dashboards gives admins the ability to check the status of any models stored inside your OpenSearch cluster. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/ml/ml-dashboard/deployed-models.png" alt="The deployed models view.">
+
+The dashboard includes the following information about the model:
+
+- **Name**: The name of the model given upon upload.
+- **Status**: The number of nodes for which the model is responsive. 
+   - When all nodes are responsive, status is **Green**.
+   - When some nodes are responsive, status is **Yellow**.
+   - When all nodes are unresponsive, status is **Red**.
+- **Model ID**: The model ID.
+- **Action**: What actions you can take with the model.
+
+As of OpenSearch 2.6, the only action available is **View Status Details**. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/ml/ml-dashboard/view-status-details.png" alt="You can view status details under actions.">
+
+When selected, the Status Details panel appears.
+
+You can see the following details inside the panel:
+
+- **Model ID**
+- **Model status by node**: The number of nodes for which the model is responsive.
```

Comment:
The number of nodes to which the model responds?

Reply:
Confirmed